lib/modules/dosomething/dosomething_signup/dosomething_signup.module

```diff
@@ -243,6 +243,10 @@ function dosomething_signup_create($nid, $uid = NULL, $source = NULL, $timestamp
     return FALSE;
   }
 
+  if (strpos($source, 'reportback') === 0) {
```

Comment:
@angaither @deadlybutter 
> why not just if `$source === 'reportback'`
To express all sources that start with reportback, ex. `reportback-foobar`?